frontend/src/components/NavigationBar.jsx

```diff
@@ -124,6 +132,20 @@ class NavigationBar extends React.Component {
                 </select>
               </FormGroup>
 
+              <FormGroup>
+                <Label for="global-config-logs-limit">Max log count</Label>
+                <select
+                  className="form-control"
+                  type="select"
```

Comment:
`select` tag don't have writable `type` attribute. You should delete it from all `select` tags.